### ISSUE TITLE
[FEATURE] Allow zipped download of open access data

### ISSUE THREAD

**Issue by Danny-dK** [UtrechtUniversity/yoda-ansible#388]
#### Is your feature request related to a problem? Please describe.
When data is openly published, there is no easy download feature to download data immediately (which is available in for example Zenodo, DANS, 4TU, Dataverse, etc). 

#### Describe the solution you'd like
Be able to download as, for example, a zip file. SURF might be implementing it as a bagit file?
<img src="https://github.com/UtrechtUniversity/yoda/assets/45395070/67467357-d5f0-4a0d-8877-205f8d31a598" width=50% height=50%>

#### Describe alternatives you've considered
Connect to the publication address in your client of choice, but can be cumbersome for some researchers.

**Comment by peer35** [comment]
I agree, but we must consider what should happen if a dataset is very large (>100GB). In that case mounting with webdav is the better option.

**Comment by Danny-dK** [comment]
True. Perhaps only up until a certain size (whatever size is acceptable). Whenever it is larger than 100GB, webdav is likely quite slow as well, I guess accessing it trough iRODS would then be better? Within the Yoda webportal at the bottom of each webpage there is the irods environment json configuration (if you click on the Yoda version). That type of configuration information could possible be shown on the public data package page? (I've actually never tried to use iCommands for a publicly available data package).

**Comment by jelletreep** [comment]
By accident opened exactly the same feature request :smile: . I think this might be a real deal breaker for smaller projects (at least for the publication step). In our project we work with a substantial amount of smaller files, and for the moment we will probably go with Dataverse instead. A size limit of e.g. 10-100 GB seems logical

**Comment by stsnel** [comment]
Internal ticket number: YDA-3735

**Comment by lwesterhof** [comment]
Implemented in https://github.com/UtrechtUniversity/yoda-ruleset/commit/9e0f05a229a86c414a8fc44c9260762d9635b5ca
